translate/lang/ro.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2007-2009 Zuza Software Foundation
+#
+# This file is part of the Translate Toolkit.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>.
+
+"""This module represents the Romanian language.
+
+.. seealso:: http://en.wikipedia.org/wiki/Romanian_language
+"""
+
+import re
+
+from translate.lang import common
+from translate.filters.checks import TranslationChecker
+from translate.filters.decorators import extraction
+
+class RomanianChecker(TranslationChecker):
+    def __init__(self, checkerconfig=None, excludefilters=None,
+                 limitfilters=None, errorhandler=None):
+        super(RomanianChecker, self).__init__(checkerconfig, excludefilters,
+                                                 limitfilters, errorhandler)
+
+    @extraction
+    def cedillas(self, str1, str2):
+        """
+        Checks if the translation strings contains an illegal cedilla character.
+        illegal_chars = [u'Ţ', u'Ş', u'ţ', u'ş']
+        Cedillas are obsoleted diacritics for Romanian and should never be used
+          where the target string is Romanian.
+        Cedilla-letters are only valid for Turkish (S-cedilla)
+          and Gagauz languages (S-cedilla and T-comma)
+          Fun fact: Gagauz is the only known language to use T-cedilla
+           :param str1: the source string
+           :param str2: the target (translated) string
+           :return: True if str2 contains a cedilla character 
+        """
+        illegal_chars = [u'Ţ', u'Ş', u'ţ', u'ş']
```

Comment:
I would define this at an upper level. Probably in the language class if possible.